src/web4/scroll_comment_into_view.js

```diff
@@ -0,0 +1,21 @@
+(async function () {
+    const urlSearchParams = new URLSearchParams(location.search);
+    const accountId = urlSearchParams.get("accountId");
+    const blockHeight = urlSearchParams.get("blockHeight");
+    if (accountId && blockHeight) {
+      for (let n = 0; n < 20; n++) {
```

Comment:
What is this `for` loop for?

Reply:
The `for` loop is for making up to 20 attempts every half second to see if the comment has become visible ( it does takes some time ), and if that is the case it will be scrolled into the view.

I thought it may be reasonable to make a finite number of attempts, in case the `accountId` and `blockHeight` parameters exists, but the page does not have the element id that can be scrolled to. This is the case for other pages than proposals.